tests/plugins/test_sharded_plugin.py

```diff
@@ -297,12 +297,24 @@ def training_step(self, batch, batch_idx):
 
 
 @RunIf(skip_windows=True, special=True, fairscale=True, min_gpus=2)
-@pytest.mark.parametrize("accelerator", ["ddp_sharded", "ddp_sharded_spawn"])
-def test_ddp_sharded_plugin_manual_optimization(tmpdir, accelerator):
+def test_ddp_sharded_plugin_manual_optimization_spawn(tmpdir):
+    # todo (sean): this test has been split out as running both tests using parametrize causes "Address in use"
```

Comment:
@SeanNaren Couldn't we just pass a different `MASTER_PORT` env variable with `accelerator` using parameterize?

Reply:
good idea! lemme try it now